release.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -xe
+
+SRC_DIR="${SRC_DIR:-$PWD}"
+
+TAG=$(git describe --tags --dirty)
+RELEASE_DIR=release-${TAG}
+#Todo include  version with flag
+BUILDFLAGS="-ldflags '-extldflags -static -X main.version=${TAG}'"
```

Comment:
Thanks for getting the `-X main.version=${TAG}` in there! At some point I'd love a way for a user to be able to pull that up too in case we're trouble shooting and need to figure out what version they have. This can be a follow on (I think there's a trellow card for it)